src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
Why do you create a separate Lottery instance for each ball?
By the way, in this case it should not be named as ball1, because it's not a Ball instance.